pkg/upgrade/upgrades/v24_1_tenant_system_tables.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 The Cockroach Authors.
```

Comment:
Shouldn't this file have a `v24_2` prefix?

Reply:
Yes, it should. Fixed.